tests/unit/test_config_manager.py

```
@@ -0,0 +1,100 @@
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)